templates/lune/partials/product/product.snip.html

```diff
@@ -0,0 +1,175 @@
+            {{/title_parts.1}}
+          </div>
+
+          <div class="lune-hero-image-wrapper z3 flex justify-center col-12">
+            {{#image}}
+              <!-- WebP image -->
+              <amp-img
+                class="lune-anim-scale-up lune-hero-image z2 col-7 md-mt0"
+                alt="{{alt}}"
+                srcset="{{webp_srcset}}"
+                width="{{width}}"
+                height="{{height}}"
+                layout="responsive">
+                  <!-- PNG Fallback when WebP is not supported -->
+                  <amp-img
+                    fallback
+                    class="lune-anim-scale-up lune-hero-image z2 col-7"
+                    alt="{{alt}}"
+                    srcset="{{fallback_srcset}}"
+                    width="{{width}}"
+                    height="{{height}}"
+                    layout="responsive">
+                  </amp-img>
+              </amp-img>
+            {{/image}}
+          </div>
+        </div>
+
+        <!-- Scroll indicator -->
+        {{> ../components/scroll-indicator.snip.html}}
+        <!-- Chapter indicators -->
+        {{> ../components/chapter-name.snip.html}}
+        {{> ../components/chapter-number.snip.html}}
+      </section>
+    {{/hero}}
+
+    {{#intro}}
+      <section class="lune-product-section lune-product-section-intro relative flex flex-column items-end justify-center p3 z2">
+        <div class="col-12 md-col-5 lg-col-4">
+          <h2 class="lune-product-section-title mb2 caps h4 md-h0">{{title}}</h2>
+
+          {{#body}}
+            <p class="lune-product-body h5 mb2">{{.}}</p>
+          {{/body}}
+
+          {{#accordion}}
+            <div class="md-hide lg-hide">
+              <amp-accordion disable-session-states>
+                <section>
+                  <h4 class="border-none"><span class="show-more h5 bold">{{more}}</span> <span class="show-less h5 bold">{{less}}</span></h4>
+                  <p class="lune-product-body h5 mb2">{{text}}</p>
+                </section>
+              </amp-accordion>
+            </div>
+
+            <p class="xs-hide sm-hide lune-product-body h5 mb2">{{text}}</p>
+          {{/accordion}}
+
+        </div>
+
+        <!-- Chapter indicators -->
+        {{> ../components/chapter-name.snip.html}}
+        {{> ../components/chapter-number.snip.html}}
+
+      </section>
+
+    {{/intro}}
+
+    {{#specs}}
+      <section class="lune-product-section relative flex flex-column justify-center p3">
+        <h2 class="lune-product-section-title caps h4 mb2 flex items-center lune-line-after md-hide lg-hide">{{title}}</h2>
+
+        <p class="lune-product-body h5 mb4 md-hide lg-hide">{{body}}</p>
+
+        <ul class="p0 m0">
+          {{#details}}
+            <li class="flex items-end pb3 md-py2">
+              <span class="lune-product-details-title h6 md-h4 caps">{{title}}</span>
+              <div class="lune-product-details-divider mx1"></div>
```

Comment:
It's tricky to achieve the same effect as an `::after` element would be treated as a child of the `lune-product-details-title`. Whereas we need this element to be a child of the `li` to make the line grow.